docs/package.json

```diff
@@ -10,17 +10,22 @@
     "build:webpack": "webpack --mode=production"
   },
   "devDependencies": {
+    "@babel/cli": "^7.18.6",
     "@babel/core": "^7.18.6",
     "@babel/preset-env": "^7.18.6",
+    "@babel/preset-react": "^7.18.6",
     "@docsearch/js": "^3.1.1",
     "autoprefixer": "^10.4.7",
     "babel-loader": "^8.2.5",
     "core-js": "^3.23.3",
     "css-loader": "^6.7.1",
-    "css-minimizer-webpack-plugin": "^4.0.0",    
+    "css-minimizer-webpack-plugin": "^4.0.0",
+    "flowbite-react": "^0.3.4",
```

Comment:
I'll later link it with the "current" build. For now, it just simplifies the test.